### PR TITLE
Tdk 1.1 first rollout fixes

### PR DESCRIPTION
**Pull request by youngca**
These are the changes applied to version 2.6.7.2, included in the TDK 1.1 release.

### CODE REVIEW

File: node_vwf.js [hunk 1]
```diff
@@ -152,8 +152,8 @@ function startVWF() {
         'destroy buffer size': Infinity, // https://github.com/socketio/socket.io/issues/1592#issuecomment-45885463; should be fixed in socket.io 1.0
     } );
     socketManager.set( 'transports', [ 'websocket' ] );
-    socketManager.set( 'heartbeat timeout', 600 );
-    socketManager.set( 'close timeout', 60 );
+    socketManager.set( 'heartbeat timeout', 3600 ); // ! hour timeout
+    socketManager.set( 'close timeout', 3600 ); // 1 hour timeout
```
Comment (davideaster) [inline node_vwf.js:156]: The close timeout should be left at 60. This is the default, so this line could be removed.

The close timeout measures the interval that the server waits after a client has disconnected unexpectedly until it gives up. The server records messages that the client has missed during this interval, so the timeout shouldn't be any longer than necessary.

File: node_vwf.js [hunk 2]
```diff
@@ -152,8 +152,8 @@ function startVWF() {
         'destroy buffer size': Infinity, // https://github.com/socketio/socket.io/issues/1592#issuecomment-45885463; should be fixed in socket.io 1.0
     } );
     socketManager.set( 'transports', [ 'websocket' ] );
-    socketManager.set( 'heartbeat timeout', 600 );
-    socketManager.set( 'close timeout', 60 );
+    socketManager.set( 'heartbeat timeout', 3600 ); // ! hour timeout
```
Comment (davideaster) [inline node_vwf.js:155]: Possibly you meant "*1* hour timeout". :-)

**Review by davideaster — CHANGES_REQUESTED** [review]

File: node_vwf.js (hunk 1 above)
Comment (youngca) [reply node_vwf.js:156]: This is what has already been delivered in TDK 1.1, so any further tweaking to the timeout parameters will have to be done in a new branch.

File: node_vwf.js (hunk 2 above)
Comment (longtinm) [reply node_vwf.js:155]: Are you planning to migrate to socket.io 1.0?  If so, please let me know about it, as I need to make the corresponding update on the HoloLens server side.

File: node_vwf.js (hunk 1 above)
Comment (youngca) [reply node_vwf.js:156]: A new task has been added to the backlog to continue testing periodically at scale and tuning as necessary.  In the meantime, this change will be merged back into the mainline to preserve what's been delivered historically.

**Pull request merged**